app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js

```diff
@@ -86,28 +57,38 @@ const LeaseUpTableContainer = ({
     <>
       <LeaseUpApplicationsFilterContainer
         preferences={preferences}
-        onSubmit={handleOnFilter}
+        onSubmit={onFilter}
         loading={loading}
+        bulkCheckboxesState={bulkCheckboxesState}
+        onClearSelectedApplications={onClearSelectedApplications}
+        onSelectAllApplications={onSelectAllApplications}
+        onBulkLeaseUpStatusChange={onLeaseUpStatusChange}
       />
       <LeaseUpApplicationsTable
         dataSet={rowsData(applications)}
         listingId={listingId}
-        onLeaseUpStatusChange={leaseUpStatusChangeHandler}
+        onLeaseUpStatusChange={onLeaseUpStatusChange}
         onCellClick={goToSupplementaryInfo}
         loading={loading}
-        onFetchData={handleOnFetchData}
+        onFetchData={onFetchData}
         pages={pages}
         rowsPerPage={rowsPerPage}
         atMaxPages={atMaxPages}
         bulkCheckboxesState={bulkCheckboxesState}
         onBulkCheckboxClick={onBulkCheckboxClick}
       />
       <StatusModalWrapper
-        {...statusModal}
-        header='Update Status'
+        alertMsg={statusModal.alertMsg}
+        isBulkChange={statusModal.isBulkChange}
+        isOpen={statusModal.isOpen}
+        loading={statusModal.loading}
+        onClose={onCloseStatusModal}
+        onSubmit={onSubmitStatusModal}
+        showAlert={statusModal.showAlert}
+        status={statusModal.status}
         submitButton='Update'
-        onSubmit={createStatusUpdate}
-        onClose={closeStatusModal}
+        title='Update Status'
+        bulkUpdateCount={statusModal.isBulkChange ? statusModal.applicationIds.length : null}
```

Comment:
nit: can this just be number of ids to change? And we decide which copy to show if it's 1 or not?

Reply:
you can use the checkbox to only update 1 status